docs/monitoring/spm-faq.md

```diff
@@ -571,24 +575,65 @@ The resulting package will contain all relevant info needed for our
 investigation. You can send it, along with short description of your
 problem, to <support@sematext.com> or contact us in chat.
 
+<b>Container setups</b>
+
+At the moment there is no diagnostics script for container setups, so various resources have to be gathered manually.
+
+Two types of agents are running in container setups and spawn from the following images:
+  - <b>sematext/agent</b> (also known as STA) - collects infrastructure data (OS and container info, metrics and events)
+  - <b>sematext/app-agent</b> (known as application agent or AA) - collects application metrics (e.g. metrics of your Elasticsearch or Kafka).
+    For each monitored service instance one such application agent container will exist. These containers are created automatically by STA.
```

Comment:
Are they only created by STA, or also managed?  If a container with a service being monitored by AA goes away, does the sematext/app-agent instance that monitored it also go away?  If so, we can say more than just "created" here.

Reply:
They are managed: service goes away, AA will go away too (done by STA). There are cases like STA uninstall, in that case any AAs will just be left hanging.

I'll adjust "created" to "created and managed".